test/e2e/framework/framework.go

```diff
@@ -220,19 +221,19 @@ func (f *Framework) Setup() error {
 		return fmt.Errorf("failed to setup qdr operator: %v", err)
 	}
 	err = f.setupQdrClusterRole()
-	if err != nil {
+	if err != nil && !HasAlreadyExistsSuffix(err) {
```

Comment:
maybe look at using `errors.IsAlreadyExists(err)` instead

Reply:
Yes, I have looked into that and that method expects the error instance to be of type  apierrors.APIStatus, but we are getting an errors.errorString instance.